### PR TITLE
Update Haitian translation

### PR DESCRIPTION
**Pull request by tstirrat**
## Description

Updated Haitian translation

## How to test

Choose Kreole from the first screen and then check all translation strings.

### CODE REVIEW

**Comment by tstirrat** [comment]
@alpita-masurkar can you check this, and @penrods can we get correct attribution name for this PR?

File: app/locales/ht.json [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
     "authorities_title": "Sous ou ka fè konfyans",
+    "choose_provider_subtitle": "Pou kapab toujou rete enfòme a ekspozisyon yo, ou bezwen enskri nan yon otorite sante.",
     "choose_provider_title": "Chwazi otorite sante",
     "commitment": "Angajman",
-    "commitment_para": "Chemen sekirize yo byen anrejistre e yo tckeke entèraksyon ak moun ki itilize posisyon ou. Done ou yo pap janm kite telefòn ou san konsantman ou.",
-    "event_history_title": "Lis evènman",
-    "export_para_1": "Tanpri fè pati  pa ou nan kanpe COVID-19 lèw pataje list pozisyon ou ak otorite lokal yo.",
-    "export_para_2": "Lokalizasyon yop rale nan fòm yon seep lis lè akm kowòdone zòn yo, okenn enfòmasyon pèsonèl pap kite transfere",
+    "commitment_para": "Chemen Sekirize byen anrejistre e tckeke entèraksyon ak moun ki itilize posisyon ou. Done ou yo pap JANM kite telefòn ou san konsantman ou.",
+    "default_news_site_name": "",
```
Comment (tstirrat) [inline app/locales/ht.json:18]: Please recommend translation here.

File: app/locales/ht.json [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
+    "event_history_subtitle": "Konprann ekspozisyo pèsonèl ou daprè enfòmasyon Otorite Santew lan pataje.",
+    "event_history_title": "Lis ekspozisyon",
+    "export_para_1": "Si ou teste pozitif pou COVID-19, tanpri fè pati pa ou nan pataje list pozisyon ou ak otorite lokal yo.",
+    "export_para_2": "Lokalizasyon yo prale nan fòm yon senp lis lè ak kowòdone zòn yo, okenn enfòmasyon anplis pap transfere",
     "home_at_risk_header": "Ou ka gen risk",
-    "home_at_risk_subtext": "Ou te kwaze chemen ak yon moun ki te teste pozitif pou COVID-19.",
+    "home_at_risk_subsubtext": "",
```
Comment (tstirrat) [inline app/locales/ht.json:24]: This can remain empty, or should we add a translation?

File: app/locales/ht.json [hunk 3]
```diff
@@ -1,31 +1,37 @@
...
+    "home_at_risk_subtext": "Daprè done disponib yo, ou te kwaze chemen ak yon moun ki te teste pozitif pou COVID-19.",
     "home_enable_location": "Aktive done lokalizasyon",
-    "home_next_steps": "Ki sa ki fè kounye a",
-    "home_no_contact_header": "kontak enkoni",
-    "home_no_contact_subtext": "Ou pa te prè pèsonn moun ki te teste pozitif pou COVID-19.",
+    "home_mayo_link_heading": "",
```
Comment (tstirrat) [inline app/locales/ht.json:27]: Need mayo link details here

File: app/locales/ht.json [hunk 4]
```diff
@@ -36,53 +42,58 @@
...
     "notification_random_data_button": "Chwazi Otorite Swen Sante",
-    "notification_title": "Ekspozisyon Profile pou 2-Semèn ",
+    "notification_title": "Ekspozisyon Pwofil pou 2 semèn",
     "notification_today": "Jodi a",
     "notification_warning_text": "Si yon Otorite Swen Sante egziste nan zòn ou an ou kapab pran abònman pou jwenn dènye enfòmasyon si gen risk ekspozisyon",
-    "notifications_exposure_format": "{{daysAgo}} jou pas, ou kwaze chemen avèk yon moun enfekte pandan {{exposureTime}} minit.",
+    "notifications_exposure_format": "{{daysAgo}} jou pase, ou kwaze chemen avèk yon moun enfekte pandan {{exposureTime}} minit.",
     "notifications_exposure_format_today": "Jodi a ou kwaze chemen avèk yon moun enfekte pandan {{exposureTime}} minit.",
     "notifications_exposure_format_yesterday": "Yè ou kwaze chemen avèk yon moun enfekte pandan {{exposureTime}} minit.",
+    "notifications_no_exposure": "",
```
Comment (tstirrat) [inline app/locales/ht.json:83]: Need "no exposure"  translattion here

File: app/locales/ht.json [hunk 5]
```diff
@@ -36,53 +42,58 @@
...
-    "settings_title": "Paramèt",
+    "settings_title": "Tablodbò",
     "share_location_data": "Pataje done lokalizasyon",
-    "show_overlap": "Klike sou yo wè done piblik la",
+    "show_overlap": "Klike sou yo wè done piblik yo",
     "team": "Ekip",
-    "team_para": "Ekip nou an konpoze de yon group ki genyen ladanl epidemyolojis, enjenyè, syantis done, evanjelizasyon vi prive dijital, pwofesè ak chèchè nan anpil gran enstitisyon nan mond lan, tank: MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young ak Link Ventures.",
-    "tested_positive_title": "Mwen teste pozitif"
+    "team_para": "Ekip nou an konpoze de yon group ki genyen ladanl epidemyolojis, enjenyè, syantis done, evanjeliS vi prive dijital, pwofesè ak chèchè nan anpil gran enstitisyon nan mond lan, tankou: MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young ak Link Ventures.",
+    "terms_of_use": "",
```
Comment (tstirrat) [inline app/locales/ht.json:95]: And one more

**Comment by alpita-masurkar** [comment]
Will try to get the missing translations.

File: app/locales/ht.json (hunk 3 above)
Comment (alpita-masurkar) [reply app/locales/ht.json:27]: looks like all the mayo links are missing in the Haitian Creole translation? I don't know if that is intentional. I will follow up on this and see if the mayo link should be there or not.

Comment (alpita-masurkar) [reply app/locales/ht.json:27]: I saw some different keys too.. there was a massive hack key which was later removed. Do the keys align between the English and the HT files?

Comment (tstirrat) [reply app/locales/ht.json:27]: Yes, this is extracted with the latest `yarn run i18n:extract` script, so it is up to date

File: app/locales/ht.json (hunk 1 above)
Comment (penrods) [reply app/locales/ht.json:18]: I think "COVID Safe Paths" is fine.

File: app/locales/ht.json (hunk 2 above)
Comment (penrods) [reply app/locales/ht.json:24]: If missing, we should use the English phrase in it's place.  Then they will complain and correct the translation.

File: app/locales/ht.json (hunk 5 above)
Comment (penrods) [reply app/locales/ht.json:95]: Will the English version show up if you don't enter anything in these?  I'm guessing not, but wanted to confirm.

Comment (tstirrat) [reply app/locales/ht.json:95]: only if I remove the key. which I can do.

**Comment by tstirrat** [comment]
I have removed the empty keys, if this is good to merge please add an approval

File: app/locales/ht.json (hunk 1 above)
Comment (tstirrat) [reply app/locales/ht.json]: Letting it fall back to english then, thanks

**Comment by alpita-masurkar** [comment]
I reached out to Gerald yesterday and asked for translation for the empty keys. Waiting for him to respond. You can merge if the keys are removed and they fall back to the English default.
Once he responds, we can quickly update the translation.

**Comment by tstirrat** [comment]
I need a re-approval, thanks

**Pull request merged**